Composer/packages/client/src/components/CreationFlow/v2/CreateBot.tsx

```diff
@@ -148,21 +148,13 @@ export function CreateBotV2(props: CreateBotProps) {
   }, []);
 
   const handleJumpToNext = () => {
-    let routeToTemplate = emptyBotKey;
-    if (option === optionKeys.createFromTemplate) {
-      routeToTemplate = currentTemplateId;
-    }
-
-    if (option === optionKeys.createFromQnA) {
-      routeToTemplate = QnABotTemplateId;
-    }
-
-    TelemetryClient.track('CreateNewBotProjectNextButton', { template: routeToTemplate });
+    TelemetryClient.track('CreateNewBotProjectNextButton', { template: currentTemplateId });
 
+    const runtimeLanguage = selectedProgLang?.props?.itemKey ? selectedProgLang.props.itemKey : csharpFeedKey;
```

Comment:
If this is a check if itemKey is null, then I think you can use:
```suggestion
    const runtimeLanguage = selectedProgLang?.props?.itemKey ?? csharpFeedKey;
```

Reply:
Good catch, I have made the change